### PR TITLE
Milestone 2 - Adelle Vo

### PR DESCRIPTION
**Pull request by adellevo**

### CODE REVIEW

**Comment by adellevo** [comment]
**Some notes on this week's task**
* did baseline task, fixed links from last week (I think), attempted EC
* still facing an error with adding instructions at a specific position — here's an example of my issue:
       * step 1: initial list size is 3, expand list by adding instruction to position 4 (works fine)
       * step 2: list size is now 4, add instruction to position 4 (unexpected behavior)
* couldn't figure out how to nicely format the JS object text

File: about.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>About Me</title>
+</head>
+<body>
+    <nav>
+        <h1><a href="/index.html">Adelle's Smoothies</a></h1>
```
Comment (ctperry0301) [inline about.html]: links are almost fixed. just add a . before the / and they will all work as expected!

<a href="/index.html"> -> <a href="./index.html">

**Review by ctperry0301 — COMMENTED** [review]
Overall the code for milestone-2 looks good and meets all the requirements. You also did the extra credit and it works really well!  I have a few notes though.

1. It looks like you accidentaly commited the node_modules folder. (thats why it says 475 files changed in github). Its a best practice to not commit your node_modules folder. You can do this by not pushing all of your files in your commits, or you can create a .gitignore file and putting the node_modules in there. [https://www.w3schools.com/git/git_ignore.asp](url)

2. Great work on the extra credit! It actually works perfectly for me, so I'm not quite sure what the unexpected behavior you mentioned is. If you think it is an issue tho, and would like help figuring this out, swing by the next office hours and we can help!

3. I wouldn't worry too much about making your json look pretty, as we will be removing this in the upcoming milestone!

4. Check the other comment I left to fix your links.

File: about.html (hunk 1 above)
Comment (adellevo) [reply about.html]: Thanks! Alternatively, would the links work if I just removed the slashes since I'm referring to a file in the same directory?

Comment (ctperry0301) [reply about.html]: I believe so ya!

**Pull request closed without merging**